iconservice/icon_inner_service.py

```diff
@@ -176,7 +178,7 @@ def _invoke(self, request: dict):
         except FatalException as e:
             self._log_exception(e, _TAG)
             response = MakeResponse.make_error_response(ExceptionCode.SYSTEM_ERROR, str(e))
-            self._close()
+            self.close()
```

Comment:
```py
    @message_queue_task
    async def close(self):
```
`close()` is async method. I'm not sure this call working properly.

Reply:
Add sync method _close() and replace self.close() to self._close()